Source/Core/DolphinNoGUI/GLFW.h

```diff
@@ -0,0 +1,211 @@
+// Copyright 2016 Dolphin Emulator Project
```

Comment:
The current X11 backend uses a lot of static variables, what would you suggest as the best way to refactor that for multiple backends?